container/raw/handler.go

```diff
@@ -200,6 +200,16 @@ func (self *rawContainerHandler) GetSpec() (info.ContainerSpec, error) {
 		if utils.FileExists(cpuRoot) {
 			spec.HasCpu = true
 			spec.Cpu.Limit = readUInt64(cpuRoot, "cpu.shares")
```

Comment:
Why add this metric only to the raw driver?
Is it possible to extract this metrics even for docker containers?